google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -1,40 +1,40 @@
 {
   "interfaces": {
-    "google.cloud.speech.v1beta1.Speech": {
+    "google.cloud.speech.v1.Speech": {
       "retry_codes": {
-        "idempotent": [
-          "DEADLINE_EXCEEDED",
-          "UNAVAILABLE"
-        ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+          "idempotent": [
+            "DEADLINE_EXCEEDED",
+            "UNAVAILABLE"
+          ],
+          "non_idempotent": [
+            "UNAVAILABLE"
+          ]
       },
       "retry_params": {
         "default": {
           "initial_retry_delay_millis": 100,
           "retry_delay_multiplier": 1.3,
           "max_retry_delay_millis": 60000,
-          "initial_rpc_timeout_millis": 60000,
+          "initial_rpc_timeout_millis": 20000,
           "rpc_timeout_multiplier": 1.0,
-          "max_rpc_timeout_millis": 60000,
+          "max_rpc_timeout_millis": 20000,
           "total_timeout_millis": 600000
```

Comment:
What does it mean to make this 20,000 milliseconds?  A request can definitely take longer than 20 seconds. 60 seconds of audio might take up 185 seconds to process worst case, though the common case is much much faster.  We should set the timeouts for the worst case.  Same question about all of the timeout_millis below.